tests/acceptance/features/bootstrap/GraphContext.php

```diff
@@ -432,7 +454,22 @@ public function removeUserFromGroup(string $groupId, string $userId, ?string $by
 	 * @throws GuzzleException
 	 */
 	public function theUserDeletesAUserUsingTheGraphAPI(string $byUser, string $user): void {
-		$this->adminDeletesUserUsingTheGraphApi($user, $byUser);
+		$userId = $this->featureContext->getUserIdByUserName($user);
+		$this->adminDeletesUserByUserIdUsingTheGraphApi($userId, $byUser);
```

Comment:
```suggestion
		$userId = $this->featureContext->getUserIdByUserName($user);
		$credentials = $this->getAdminOrUserCredentials($byUser);
		$this->featureContext->setResponse(
			GraphHelper::deleteUserByUserId(
				$this->featureContext->getBaseUrl(),
				$this->featureContext->getStepLineRef(),
				$credentials["username"],
				$credentials["password"],
				$userId
			)
		);
```

Reply:
why we have to duplicate codes like this? We can simply return response and setResponse in When step